src/buffer/out/textBuffer.cpp

```diff
@@ -2241,7 +2242,8 @@ HRESULT TextBuffer::Reflow(TextBuffer& oldBuffer,
         // Loop through every character in the current row (up to
         // the "right" boundary, which is one past the final valid
         // character)
-        for (short iOldCol = 0; iOldCol < iRight; iOldCol++)
+        short iOldCol = 0;
+        for (; iOldCol < iRight; iOldCol++)

```

Comment:
And we couldn't just set `iRight` to the std::min(oldBufferWidth, newBufferWidth) and let it dutifully copy the empty spaces with the color instead?

Reply:
.... maybe

Reply:
maybe not. If we do that, then we'll copy the blank spaces from the old buffer onto a new line the smaller new buffer........


but we could do 
```c++
std::min(oldBufferWidth, std::max(newBufferWidth, iRight))
```

copy all the chars up to the minimum of { the old buffer's width, ( the max of { the new buffers width or the old buffer's rightmost char } ) }. So if we resize narrower, the new buffer will be smaller. if the right 


* narrower, iRight > newBufferWidth: we want to copy all up to iRight  (or oldBufferWidth)
* narrower, iRight < newBufferWidth: we want to copy all up to newBufferWidth......................................




I couldn't get this right without copying extra spaces 🤷

Reply:
Yeah that's fine. Thanks.